interactions/client/models/component.py

```diff
@@ -266,7 +265,8 @@ class ActionRow(ComponentMixin):
         An ActionRow may also support only 1 text input component
         only.
 
-    The structure for an action row: ::
+    The structure for an action row:
+    ::
```

Comment:
```suggestion
    ..code-block:: python
```

Reply:
btw my way also works (pycharm said it works)